powder_reduction.py

```diff
@@ -163,9 +168,11 @@ def powder_reduction(sample='sample.nxs',
         del correction.coords['sample_position']
         del correction.coords['position']
 
-        sample_lambda /= correction
+        correction_rebin = sc.rebin(correction, 'wavelength', edges_lambda)
 
-        del correction
+        sample_lambda /= correction_rebin
+
+        del correction, correction_rebin
```

Comment:
Why remove these explicitly? Don't they go out of scope at the end of the `if`?

Reply:
It will be removed